src/pybind/mgr/cephadm/inventory.py

```diff
@@ -172,9 +173,9 @@ def __init__(self, mgr):
         self.osdspec_previews = {}     # type: Dict[str, List[Dict[str, Any]]]
         self.networks = {}             # type: Dict[str, Dict[str, List[str]]]
         self.last_device_update = {}   # type: Dict[str, datetime.datetime]
-        self.daemon_refresh_queue = [] # type: List[str]
-        self.device_refresh_queue = [] # type: List[str]
-        self.osdspec_previews_refresh_queue = [] # type: List[str]
+        self.daemon_refresh_queue = []  # type: List[str]
+        self.device_refresh_queue = []  # type: List[str]
+        self.osdspec_previews_refresh_queue = []  # type: List[str]
```

Comment:
same as https://github.com/ceph/ceph/pull/36523#discussion_r476443833?

Reply:
you're right and I was wrong. wdyt of merging it anyway?

Reply:
eh there's not really a right way, just curious if you wanted to keep the alignment. I'm aok with this.